main/Smartsheet/Api/Models/DataWrapper.cs

```diff
@@ -0,0 +1,71 @@
+﻿//    #[license]
+//    SmartsheetClient SDK for C#
+//    %%
+//    Copyright (C) 2014 SmartsheetClient
+//    %%
+//    Licensed under the Apache License, Version 2.0 (the "License");
+//    you may not use this file except in compliance with the License.
+//    You may obtain a copy of the License at
+//        
+//            http://www.apache.org/licenses/LICENSE-2.0
+//        
+//    Unless required by applicable law or agreed To in writing, software
+//    distributed under the License is distributed on an "AS IS" BASIS,
+//    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+//    See the License for the specific language governing permissions and
+//    limitations under the License.
+//    %[license]
+
+using System;
+using System.Collections.Generic;
+
+namespace Smartsheet.Api.Models
+{
+
+	/// <summary>
+	/// A wrapper object used to Wrap the data that comes back from the API. <br />
+	/// It holds the paging info as well as a List of objects of the specified type.
+	/// </summary>
+	public class DataWrapper<T>
```

Comment:
I'd recommend something a little clear on what it is, like "PaginatedResult", since this is used exclusively with paginated requests. DataWrapper is pretty vague and doesn't communicate very well to the developer what it's intent is.